platform/android/src/native_map_view.cpp

```diff
@@ -257,12 +257,12 @@ void NativeMapView::moveBy(jni::JNIEnv&, jni::jdouble dx, jni::jdouble dy, jni::
 void NativeMapView::jumpTo(jni::JNIEnv&, jni::jdouble angle, jni::jdouble latitude, jni::jdouble longitude, jni::jdouble pitch, jni::jdouble zoom) {
     mbgl::CameraOptions options;
     if (angle != -1) {
-        options.angle = -angle * util::DEG2RAD;
+        options.angle = angle;
```

Comment:
Was removing `-` your intention here (and in the other changesets in this file)

Reply:
Yes - that's only needed when converting bearing degrees into radians.

Reply:
Specifically, the minus sign was being used to accommodate for the fact that the bearing is taken counterclockwise.